pkgs/by-name/ox/oxker/package.nix

```
@@ -2,19 +2,22 @@
   lib,
   fetchCrate,
   rustPlatform,
+  nix-update-script,
 }:
 
 rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

Could you remove the `rec`?